README.md

```diff
@@ -70,6 +70,18 @@ Since 1.3.0:
 * `quarkus.tools.starter.showExtensionDescriptions`: Determines whether to show the Quarkus extension descriptions when selecting Quarkus extensions. Default is `true`.
 * `quarkus.tools.validation.value.severity`: Validation severity for property values for Quarkus/MicroProfile `*.properties` files. Default is `error`.
 
+### <mp-rest-client-class>/mp-rest/providers/<mp-rest-provider-class>/priority
+
+By waiting a fix for[this issue](https://github.com/redhat-developer/quarkus-ls/issues/203), the mp rest `<mp-rest-client-class>/mp-rest/providers/<mp-rest-provider-class>/priority` reports an unknown error. 
+
+To avoid having this error, you must configure this settings:
+
+```json
+"quarkus.tools.validation.unknown.excluded": [
+    "*/mp-rest/providers/*/priority"
+]
+```
+
```

Comment:
In terms of formatting, how about something like:

How about something like:

---
### **Note for MicroProfile Rest Client properties**:

Due to [this issue](https://github.com/redhat-developer/quarkus-ls/issues/203), the MP Rest property: `<mp-rest-client-class>/mp-rest/providers/<mp-rest-provider-class>/priority` reports an unknown error. 

To avoid having this error, you must configure the following `settings.json`:

```json
"quarkus.tools.validation.unknown.excluded": [
    "*/mp-rest/providers/*/priority"
]
```
---

Reply:
Thank a lot @xorye for your suggestion. Done.